tests/test_model.py

```diff
@@ -751,6 +753,60 @@ def test_generate(attention_impl, device, alibi):
             generation_with_left_padding[:, 6:])
 
 
+@pytest.mark.gpu
+@pytest.mark.parametrize('world_size', [1, 2])
+def test_generate_with_device_map(tmp_path, world_size):
```

Comment:
Should we parameterize this with
```
@pytest.mark.parametrize('use_cache', [False, True])
```

(I think it would be passed to `MPTConfig`)